doc/adr/0012-rewards.md

```diff
@@ -0,0 +1,44 @@
+# Rewards
+
+Date: 2022-06-01
+
+## Status
+
+Proposed
+
+## Context
+
+Facilitators get rewarded in DAG currency for participating in consensus rounds.
+We need to figure out the algorithm for minting/sending rewards to facilitators. Algorithm
+should take dynamic network speed into account to determine the size of rewards per node.
```

Comment:
Then this part could be:
"
The rewards model requires network speed (snapshots per min) as an input. In previous implementation this value was hardcoded. Currently, we need to figure out the algorithm for determining the network speed dynamically and deterministically, so then the reward model can be calculated locally as a pure function, assuming all inputs will be equal on each node.
"